src/mono/wasm/build/WasmApp.targets

```diff
@@ -152,6 +152,7 @@
     <WasmStripILAfterAOT Condition="'$(WasmStripILAfterAOT)' == ''">false</WasmStripILAfterAOT>
 
     <WasmRuntimeAssetsLocation Condition="'$(WasmRuntimeAssetsLocation)' == ''">_framework</WasmRuntimeAssetsLocation>
+    <BlazorCacheBootResources Condition="'$(BlazorCacheBootResources)' == ''">true</BlazorCacheBootResources>
```

Comment:
We should use a non-blazor specific property name in WasmApp.targets .

Reply:
We are using the same property in both Wasm SDK and Blazor SDK. Maybe we should introduce alternative names for properties integrated from Blazor SDK and replace Blazor with Wasm in their name